styles/base.css

```diff
@@ -42,40 +54,106 @@ a {
     transition: 250ms;
 }
 
-a:hover, a:focus {
+a:hover,
+a:focus {
     color: var(--green);
 }
 
 ul.no-bullets {
+    padding-left: 0;
+}
+
+ul.no-bullets li {
     list-style-type: none;
     padding: 0;
-    margin: inherit 0 0;
+    margin: 16px 0;
+}
+
+@media (min-width: 992px) {
+    ul.no-bullets li {
+        margin: inherit 0 0;
+    }
 }
 
 li.certs {
     padding-bottom: 0.75rem;
 }
 
-.monospace, h1, h2, h3, h4, h5, h6 {
+/*#region Font Family */
+
+.monospace,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
     font-family: 'Source Code Pro Bold', 'Courier New', Courier, monospace;
 }
```

Comment:
Looking at the Style Guide, https://www.figma.com/file/2gq3WJFdRFDefyHcgFMUGt/Current-compiler.la?node-id=326-322&t=qbG8SaSrYZENtOYA-0, only H1 and H2 are Source Code Pro.

Reply:
Actually - Just checked the rest of the repo. The repo only uses h1 and h2 so far, so let's just remove h3-h6 here.